proto/minder/v1/minder.proto

```diff
@@ -1775,13 +1775,25 @@ message ListEvaluationResultsRequest {
     // an empty label_filter.
     oneof profile_selector {
         // ID can contain either a profile name or an ID
-        string profile = 2;
+        string profile = 2 [
+            (buf.validate.field).string = {
+                pattern: "^([A-Za-z][-a-zA-Z0-9_]*|[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12})$",
```

Comment:
I think this might be easier to read using character classes:

```suggestion
                pattern: "^([[:alpha:]][-[:word:]]*|[[:xdigit:]]{8}-[[:xdigit:]]{4}-[[:xdigit:]]{4}-[[:xdigit:]]{4}-[[:xdigit:]]{12})$",
```

However, if you simply changed this to:

```suggestion
                pattern: "^([[:alnum:]][-[:word:]]*|$",
```

You'd cover both names and UUIDs with one pattern.

Reply:
Thanks! 👍 I'll apply the change myself though otherwise you'll not be able to approve it.